tests/lax_numpy_test.py

```diff
@@ -1152,6 +1152,23 @@ def testPolySub(self, a_shape, b_shape, dtype):
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "a_shape={} , order={}".format(
+          jtu.format_shape_dtype_string(a_shape, dtype),
+          order),
+       "dtype": dtype, "a_shape": a_shape, "order" : order}
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes
+      for order in range(8)))
+  def testPolyDer(self, a_shape, order, dtype):
+    rng = jtu.rand_default(self.rng())
```

Comment:
Our more complete set of internal tests is seeing test failures related to infinities at high order. The issue is when you combine low-precision dtypes (such as float16) with high-order derivatives of large polynomials (which multiply the largest coefficient by `N! / (N-m)!` for the `m`th derivative of a polynomial of order `N`), the resulting coefficients overflow the dtype's range.

I would recommend either reducing the maximum order to, say, 4, or skipping the test for `float16`/`bfloat16`